keyboards/acheron/lasgweloth/readme.md

```diff
@@ -0,0 +1,29 @@
+# Acheron Aχξρων 60-SM-S-STM32-MX-TH-WI (codename "Lasgweloth") QMK firmware
+
+<p align="center">
+  <img align="middle" src="https://raw.githubusercontent.com/Gondolindrim/acheronLibrary/master/graphics/acheronLong.png"  width="400"> 
+</p>
+
+## Introduction
+
+This is the QMK firmware repository for the Lasgweloth PCB, updated until [pre-revision Alpha](https://github.com/AcheronProject/Lasgweloth/releases/).
+
+The Lasgweloth PCB is an Open-Hardware guidelines compliant PCB which files can be found at [this link](https://github.com/AcheronProject/Lasgweloth). Its designer and maintainer is [Gondolindrim](https://github.com/Gondolindrim).
+
+## Layouts
+
+The possible layouts consist of a default or 7U bottom row, with possibility of using split backspace, split right shift and stepped caps lock.
+
+## PCB Documentation
+
+See the [AcheronDocs](https://gondolindrim.github.io/AcheronDocs/lasgweloth/intro.html) page for the Lasgweloth full documentation. You can also check the KiCad PCB files at the [GitHub repository](https://github.com/AcheronProject/Lasgweloth).
```

Comment:
404's

Reply:
The Acheron Project website was wrong link. The github links are fine, it's just that the repo is not public yet but will be once I ship the GB PCBs